docs/api-reference/loaders-storage-targets/lake-loader/maintenance/_iceberg.md

```diff
@@ -0,0 +1,64 @@
+The [Iceberg documentation](https://iceberg.apache.org/docs/latest/maintenance/) makes recommendations for running regular maintenance jobs to get the best performance from your lake.  This guide expands on those recommendations specifically for your Snowplow events lake.
+
+The Snowplow Lake Loader does not automatically run the maintenance tasks described below.
+
+## Expire snapshots
+
+We recommend that you schedule the `expireSnapshots` Iceberg action to run once per day.
+
+The Snowplow Lake Loader is a continuously-running streaming loader, so it creates new snapshots very frequently, each time it commits more events into the lake.  So it is especially important to manage the total number of snapshots held by the Iceberg metadata.
+
+There are two benefits of expiring snapshots in your Snowplow lake:
+
+1. The snapshot metadata files can be much smaller, because the list of metadata files to track is much reduced.  This reduces the overhead of creating a new snapshot file, and so it improves the performance of the Lake Loader when committing new events into the lake.  This becomes especially important as your lake grows in size over time.
```

Comment:
```suggestion
1. The snapshot metadata files can be much smaller, because the list of metadata files to track is much smaller.  This reduces the overhead of creating a new snapshot file, and thus improves the performance of the Lake Loader when committing new events into the lake.  This becomes especially important as your lake grows in size over time.
```